funnel/models/proposal.py

```diff
@@ -477,6 +477,11 @@ def all_public(cls):
 add_search_trigger(Proposal, 'search_vector')
 
 
+@Proposal.views('video', cached_property=True)
+def proposal_video_property(obj):
+    return _video_property(obj)
```

Comment:
Why does this need a wrapper? `Proposal.views('video', cached_property=True)(_video_property)` does the same thing. Also, why is a view function being imported from a model?